### PR TITLE
Mention download counts being stored

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

**Comment by sigurdm** [comment]
cc @mit-mit

File: doc/help-scoring.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ package page.
 To view packages you have liked, use the **My pub.dev > My liked packages** menu
 option.
 
+## Download counts
+
```
Comment (mit-mit) [inline doc/help-scoring.md:30]: Suggested rewrite:

Pub.dev tracks and stores how many times each version of each package gets downloaded. This is used to display package usage metrics.

**Review by szakarias — APPROVED** [review]
LGTM with Michael's rewrite.

File: doc/help-scoring.md (hunk 1 above)
Comment (sigurdm) [reply doc/help-scoring.md:30]: Done

**Review by mit-mit — APPROVED** [review]

**Review by jonasfj — APPROVED** [review]

**Pull request merged**